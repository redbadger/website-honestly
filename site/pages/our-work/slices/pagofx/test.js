import React from 'react';
import { shallow } from 'enzyme';

import PagoFxSlide from './index';

describe('pages/our-work/case-study/slices/pride', () => {
  it('renders correctly', () => {
    const rendered = shallow(<PagoFxSlide />);
    expect(rendered).toMatchSnapshot();
  });
});
