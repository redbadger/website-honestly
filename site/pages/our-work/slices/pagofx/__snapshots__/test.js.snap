// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/our-work/case-study/slices/pride renders correctly 1`] = `
<Container
  to="pagoFxCaseStudy"
>
  <Text
    layout="leftCol"
  >
    <Header>
      Delivering an end-to-end international transfers product in just 18 months 
    </Header>
    <Description>
      Learn how Red Badger helped Santanderâ€™s PagoFX team to design, build and deliver a best-in-class payments service.
    </Description>
    <ReadMore />
  </Text>
  <Image
    alt="iPhone showing Pago FX application "
    src="pagofx.jpg"
  />
</Container>
`;
